src/net/sourceforge/kolmafia/request/UseSkillRequest.java

```diff
@@ -76,6 +76,7 @@ public class UseSkillRequest extends GenericRequest implements Comparable<UseSki
     "Bowl Full of Jelly",
     "Eye and a Twist",
     "Chubby and Plump",
+    "Canticle of Carboloading"
```

Comment:
Probably put this in adifferent position to the end of the list, because it's actually very old. If possible.